src/transform/namer.ts

```diff
@@ -67,17 +67,22 @@ export async function namer(session: Session<CodeModel>) {
 
   // pascal-case and capitzalize acronym operation groups and their operations
   for (const group of values(model.operationGroups)) {
-    const details = <Language>group.language.go;
-    details.name = capitalizeAcronyms(pascalCase(details.name));
-    // we don't call GetEscapedReservedName here since any operation group that uses a reserved word will have 'Operations' attached to it
-    details.clientName = `${details.name}Operations`;
-    // this sets Operations as the default name for operation groups that don't specify a group name
-    if (length(details.name) === 0) {
-      details.name = `Operations`;
+    const groupDetails = <Language>group.language.go;
+    // use the swagger title as the default name for operation groups that don't specify a group name
+    if (groupDetails.name.length === 0) {
+      groupDetails.name = pascalCase(session.model.info.title);
```

Comment:
Isn't the call to pascalCase repetitive here since we call it directly below as well?

Reply:
Yep, will fix.